#real_world: false

figure_finder:
  ros__parameters:
    figure_collector:
      group_num_instances: 7
      group_dist_thresh: 1.5
      use_local_coords: true
    figure_operations_config_file: config/figure_operations.yaml  
    color_detection_config_file: config/color_detection_sim.json 

detection:
  ros__parameters:
    nn_model_path: nn_models/best_barrel_unity.pt
    detector: yolo

tf_start_pose:
   ros__parameters:
    drone_z_offset: 0.0

#additional settings
#upload_data: true